src/ucp/api/ucp.h

```diff
@@ -4595,6 +4597,18 @@ typedef struct ucp_ep_attr {
      * UCS_ERR_NOT_CONNECTED will be returned.
      */
     struct sockaddr_storage remote_sockaddr;
+
+     /**
+      * Available debug info in string format which fits into
+      * @ref ucp_ep_attr_t::debug_str_max.
+      */
+    char                    *debug_str;
```

Comment:
same as in request:
debug_string
debug_string_size

Reply:
ok, will fix

Reply:
BTW, as I can see ucp_request_query API is not clear about null-terminator, is debug_string_size should include it or not? buffer size != string length
but for example IIRC `std::string::length() == std::string::size() ` without '\0' but this is not size of allocated c-style-buffer

Reply:
IMO we should NEVER fill beyond the specified string length. If it's 0 - do not fill anything